src/main/java/jenkins/bouncycastle/api/PEMEncodable.java

```diff
@@ -153,7 +174,7 @@ public static PEMEncodable decode(@Nonnull String pem, @Nullable final char[] pa
                 throw new IOException("Could not parse PEM, only key pairs, private keys, public keys and certificates are supported");
             }
 
-            JcaPEMKeyConverter kConv = new JcaPEMKeyConverter().setProvider("BC");
+            JcaPEMKeyConverter kConv = new JcaPEMKeyConverter().setProvider(BOUNCY_CASTLE_PROVIDER.getName());
```

Comment:
is `setProvider(Provider)` not available in both versions?

```suggestion
            JcaPEMKeyConverter kConv = new JcaPEMKeyConverter().setProvider(BOUNCY_CASTLE_PROVIDER);
```

Reply:
Yes, it is. Merging your suggestion now :-)